.github/actions/upload-benchmark-results/action.yml

```diff
@@ -9,6 +9,8 @@ inputs:
   # TODO (huydhn): Use this to gate the migration to oss_ci_benchmark_v3 on S3
   schema-version:
     default: 'v2'
+  github-token:
+    default: ''
```

Comment:
this is needed for v3 right, maybe we can have a check that this is given if v3 is set?

Reply:
Sound good.  I'm wondering if I could leave the job id optional even for v3, but then it would complicate thing like writing query joining with workflow_job.  It seems easier to make this mandatory for v3